docs/pages/access-controls/guides/headless-login.mdx

```diff
@@ -0,0 +1,237 @@
+---
+title: Headless Login (Preview)
+description: Headless Login
+---
+
+## Introduction
+
+<Admonition type="warning" title="Preview">
+  Headless login is currently in Preview mode.
+</Admonition>
+
+Headless login provides Teleport users with a secure way to authenticate with WebAuthn from a remote machine. This enables use cases which usually are not compatible with WebAuthn features, such as:
```

Comment:
This description indicates an advantage of headless login, but I'm left wondering what the nature of the feature is. 

Usually, I interpret "headless" as meaning "without a GUI". Yet users could always log in to Teleport without a GUI via `tsh`, so I think we need a more explicit description of how this feature works/what it is to avoid misconceptions.

Would it be more accurate to say this is "Headless WebAuthn Login", or simply "Headless WebAuthn"?

Reply:
@jakule I think I failed to ping you in the comment above as I accidentally included my response in the quoted section.

Reply:
@r0mant WDYT?

Reply:
@klizhentas @xinding33 Can you please help use decide on a final product name? Then I can make sure we have the name standardized across user facing components before we release the feature next week.

Some potential options are:
 - Headless Login
 - Headless Authentication
 - Headless WebAuthn
 - Headless SSO